spec/std/isa/csr/vsstatus.yaml

```
@@ -57,6 +58,26 @@ fields:
       return (VUXLEN == 3264) ? CsrFieldType::RW : CsrFieldType::RO;
     reset_value(): |
       return (VUXLEN == 3264) ? UNDEFINED_LEGAL : VUXLEN;
+  SDT:
+    location: 41
+    base: 64
+    description: |
+      *Supervisor Disable Trap*
+
+      Written to 1 when entering VS-mode from an exception/interrupt.
+      When returning via an SRET instruction, the bit is written to 0.
+      On reset is set to 0.
+      When vsstatus.SDT=1, direct write by CSR instruction cannot set vsstatus.SIE to 1, if not written together.
+
+      This field is read-only zero if `henvcfg.DTE` is zero.
+    definedBy: Ssdbltrp
+    type: RW-H
+    reset_value: UNDEFINED_LEGAL
```

Above, it says:
> On reset is set to 0.

So, implement that here:
```suggestion
    reset_value: 0
```